java/libphonenumber/src/com/google/i18n/phonenumbers/MetadataManager.java

```diff
@@ -130,10 +104,9 @@ private static void loadAlternateFormatsMetadataFromFile(int countryCallingCode)
       // that they are present, by checking the map of available data first.
       throw new IllegalStateException("missing metadata: " + fileName);
     }
-    PhoneMetadataCollection alternateFormatData =
-        loadMetadataAndCloseInput(source, DEFAULT_BUFFER_SIZE);
-    for (PhoneMetadata metadata : alternateFormatData.metadata) {
-      callingCodeToAlternateFormatsMap.put(metadata.countryCode, metadata);
+    PhoneMetadataCollection alternateFormats = loadMetadataAndCloseInput(source);
```

Comment:
can we change this back to alternateFormatData? same below for shortNumberMetadata; alternatively, make sure you make the same changes in our copy.